project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -417,7 +417,7 @@ public void testLoadingBuildOptionsFromToml() {
 
         // Verify expected default buildOptions
         Assert.assertTrue(project.buildOptions().skipTests());
-        Assert.assertTrue(project.buildOptions().observabilityIncluded());
+        Assert.assertFalse(project.buildOptions().observabilityIncluded());
```

Comment:
Same as other assertion. (can revert this by reverting `project-api/project-api-test/src/test/resources/projectWithBuildOptions/Ballerina.toml`)